components/reply-popup.jsx

```diff
@@ -0,0 +1,79 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import Avatar from './avatar.jsx';
+import ThemeContext from './theme-context.jsx';
+
+/**
+ * React component for an reply popup
+ * 
+ * @param   {string} homeserver - The homeserver URL
+ * @param   {object} mxEvent - The event object
+ * @param   {object} client - The matrix client object
+ * @param   {func} replyTo - Callback for setting reply
+ */
+export default class ReplyPopup extends PureComponent {
+    static propTypes = {
+        homeserver: PropTypes.string.isRequired, // Homeserver URL
+        mxEvent: PropTypes.object.isRequired, // Event object
+        client: PropTypes.object.isRequired, // Client object
+        replyTo: PropTypes.func.isRequired // Callback for setting reply
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.unsetReply = this.unsetReply.bind(this);
+    }
+
+    unsetReply() {
```

Comment:
This needs to be called after message send, otherwise the reply box stays.

Reply:
Oh right, I'll fix that.